libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -166,51 +168,73 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
 AP_ADSB::AP_ADSB()
 {
     AP_Param::setup_object_defaults(this, var_info);
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
     if (_singleton != nullptr) {
         AP_HAL::panic("AP_ADSB must be singleton");
     }
+#endif
     _singleton = this;
+
+#ifdef ADSB_STATIC_CALLSIGN
```

Comment:
This looks like new code?

I do note it is in a separate commit, and its a pretty minor addition.

Reply:
yes, this is new